src/app/watchlist/models/editWatchModel.go

```diff
@@ -0,0 +1,6 @@
+package models
+
+type EditWatchlist struct {
```

Comment:
keep all model names as APINameRequest if it's a request model.
i.e name should be "EditWatchlistRequest" here